specs/endpoint-security.spec.yml

```diff
@@ -14,6 +14,7 @@ inputs:
     proxied_actions: &proxied_actions
       - UNENROLL
       - UPGRADE
+      - MIGRATE
```

Comment:
I would expect here to proxy the action to endpoint as we do [here](https://github.com/elastic/elastic-agent/blob/05f62c2a8e9766c0d513c0791c386d38d67276c4/internal/pkg/agent/application/actions/handlers/handler_action_unenroll.go#L80) for `unenroll` and [here](https://github.com/elastic/elastic-agent/blob/05f62c2a8e9766c0d513c0791c386d38d67276c4/internal/pkg/agent/application/actions/handlers/handler_action_upgrade.go#L65) for `upgrade`. Is this migrate action being proxied to endpoint somewhere else that I am missing?